server/routes.py

```diff
@@ -21,6 +29,45 @@ def products():
 def index():
     return jsonify({'success': True}), 200
 
+@app.route('/2fa', methods=['POST'])
+def twoFA():
+    email = request.form['email']
+
+    user = User.query.filter(User.email == email).first()
+
+    print(user.session_token, request.form['session_token'])
```

Comment:
You should remove print statements when pushing code, so the user's session token does not display.